todo-frontend/src/services/user.service.js

```diff
@@ -5,14 +5,20 @@ axios.interceptors.request.use(
       if (token != '') {
         config.headers['Authorization'] = 'Bearer ' + token;
       }
-  
       return config;
-    }, 
-  
-    (error) => {
-      return Promise.reject(error);
-    }
-);
+    }, (err)=> {
+        return err;
+    });
+
+//provera da li je code 401
+axios.interceptors.response.use((response) => {
+    return response;
+}, (err) => {
+    if(err.response.status == 401) localStorage.setItem('token', '');
```

Comment:
kul to je to
samo jos kad tamo izbacis localstorage u service, odavde mozes mozda isto da dispatchujes akciju da ona dalje ishendla sklanjanje tokena i sta vec treba